src/locales/en/submission/detail.json

```diff
@@ -1,4 +1,8 @@
 {
+  "archive": "Copy the submission in the malware archive",
```

Comment:
Copy the submission **in** the malware archive → Copy the submission **into** the malware archive ?